contrib/log_correlation/stackdriver/src/main/java/io/opencensus/contrib/logcorrelation/stackdriver/OpenCensusTraceLoggingEnhancer.java

```diff
@@ -24,35 +24,114 @@
 import io.opencensus.trace.TraceId;
 import io.opencensus.trace.Tracer;
 import io.opencensus.trace.Tracing;
+import java.util.logging.LogManager;
 
 /**
  * Stackdriver {@link LoggingEnhancer} that adds OpenCensus tracing data to log entries.
  *
  * <p>This feature is currently experimental.
  */
-
-// TODO(sebright): Allow configuring the LoggingEnhancer with Java properties. One property should
-// control the types of spans to correlate, with these three choices:
-// - off
-// - only add tracing data for sampled spans
-// - add tracing data for all spans
 @ExperimentalApi
 public final class OpenCensusTraceLoggingEnhancer implements LoggingEnhancer {
-  private static final String SAMPLED_KEY = "sampled";
+  private static final String SAMPLED_LABEL_KEY = "sampled";
+  private static final SpanSelection DEFAULT_SPAN_SELECTION = SpanSelection.SAMPLED_SPANS;
```

Comment:
are all trace exported typically? OR only the sampled one? 
If it is all then default should be ALL_SPANS.

Reply:
I changed the default to ALL_SPANS.